python/hail/docs/methods/import.rst

```diff
@@ -0,0 +1,30 @@
+Import
```

Comment:
let's combine import/export. ImportVCF and exportVCF should be on the same page, yeah?